File: app/src/main/java/net/mieczkowski/yelpbusinessexample/models/business/BusinessCoordinates.java
package net.mieczkowski.yelpbusinessexample.models.business;

import android.os.Parcel;
import android.os.Parcelable;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.Table;

import net.mieczkowski.yelpbusinessexample.database.MyDatabase;
import net.mieczkowski.yelpbusinessexample.database.PrimaryKeyModel;

/**
 * Created by Josh Mieczkowski on 10/19/2017.
 */

@Table(database = MyDatabase.class)
public class BusinessCoordinates extends PrimaryKeyModel {

    @Column
    @JsonProperty("latitude")
    double latitude;

    @Column
    @JsonProperty("longitude")
    double longitude;

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public BusinessCoordinates() {
    }

}

File: dal/src/main/java/net/mieczkowski/dal/services/businessLookupService/models/BusinessCoordinates.kt
package net.mieczkowski.dal.services.businessLookupService.models

import com.fasterxml.jackson.annotation.JsonProperty
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.Table
import net.mieczkowski.dal.cache.LocalDatabase
import net.mieczkowski.dal.cache.models.PrimaryKey

/**
 * Created by Josh Mieczkowski on 9/11/2018.
 */
@Table(database = LocalDatabase::class)
class BusinessCoordinates: PrimaryKey() {

    @Column
    @JsonProperty("latitude")
    var latitude: Double = 0.0

    @Column
    @JsonProperty("longitude")
    var longitude: Double = 0.0
}